core/templating/template_helpers.go

```diff
@@ -135,34 +135,130 @@ func (t templateHelpers) isBool(s string) bool {
 	return err == nil
 }
 
-func (t templateHelpers) isGreaterThan(valueToCheck, minimumValue string) bool {
-	num1, err := strconv.ParseFloat(valueToCheck, 64)
+// func isGreaterThan(valueToCheck, minimumValue string) bool {
+// 	num1, err := strconv.ParseFloat(valueToCheck, 64)
+// 	if err != nil {
+// 		return false
+// 	}
+// 	num2, err := strconv.ParseFloat(minimumValue, 64)
+// 	if err != nil {
+// 		return false
+// 	}
+// 	return num1 > num2
+// }
+
+// func (t templateHelpers) isGreaterThan(valueToCheck, minimumValue string) bool {
+// 	return isGreaterThan(valueToCheck, minimumValue)
+// }
+
+// func isGreaterThanOrEqual(valueToCheck, minimumValue string) bool {
+// 	num1, err := strconv.ParseFloat(valueToCheck, 64)
+// 	if err != nil {
+// 		return false
+// 	}
+// 	num2, err := strconv.ParseFloat(minimumValue, 64)
+// 	if err != nil {
+// 		return false
+// 	}
+// 	return num1 >= num2
+// }
+
+// func (t templateHelpers) isGreaterThanOrEqual(valueToCheck, minimumValue string) bool {
+// 	return isGreaterThan(valueToCheck, minimumValue)
+// }
+
+// func isLessThan(valueToCheck, maximumValue string) bool {
+// 	num1, err := strconv.ParseFloat(valueToCheck, 64)
+// 	if err != nil {
+// 		return false
+// 	}
+// 	num2, err := strconv.ParseFloat(maximumValue, 64)
+// 	if err != nil {
+// 		return false
+// 	}
+// 	return num1 < num2
+// }
+
+// func (t templateHelpers) isLessThan(valueToCheck, maximumValue string) bool {
+// 	return isLessThan(valueToCheck, maximumValue)
+// }
+
+// func isLessThanOrEqual(valueToCheck, maximumValue string) bool {
+// 	num1, err := strconv.ParseFloat(valueToCheck, 64)
+// 	if err != nil {
+// 		return false
+// 	}
+// 	num2, err := strconv.ParseFloat(maximumValue, 64)
+// 	if err != nil {
+// 		return false
+// 	}
+// 	return num1 <= num2
+// }
+
+// func (t templateHelpers) isLessThanOrEqual(valueToCheck, maximumValue string) bool {
+// 	return isLessThan(valueToCheck, maximumValue)
+// }
+
+// func (t templateHelpers) isBetween(valueToCheck, minimumValue, maximumValue string) bool {
+// 	return t.isGreaterThan(valueToCheck, minimumValue) && t.isLessThan(valueToCheck, maximumValue)
+// }
+
+//########################
+
+// Comparison function type that takes two float64 values and returns a boolean.
```

Comment:
it's better to remove the code if we don't need it.

Reply:
Whoops, yes I will delete

Reply:
Done